.github/ISSUE_TEMPLATE/---report-a-bug.md

```diff
@@ -26,13 +26,17 @@ If you can, add screenshots to show the problem. If you need to show a code exam
 ## 🖥 Desktop
  - OS: [e.g. macOS]
  - Browser: [e.g. chrome, safari]
- - Version: [e.g. 22]
+ - Browser Version: [e.g. 22]
+- Website Version [eg. V1, V2]
```

Comment:
```suggestion
 - Website Version: [e.g. V1, V2]
```
Same on line 38

Also not sure 'version' needs to be capitalised - but doesn't really matter either way

Reply:
I think the capitalisation of V1, V2 is ok as on the site it is capitalised also, therefore makes sense to leave it as it is.

Reply:
I was thinking just the whole word 'Version', but it's not really a huge deal - happy either way. I agree that V1, V2 should be capitalised though :)

Reply:
I agree that version shouldn't be capitalised if it's not the first word i.e. "Browser version", "Website version"